Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -424,5 +430,17 @@ void UpdateOrientation()
 				}
 			}
 		}
+
+		void UpdateHorizontalScrollBarVisibility()
+		{
+			if (_hScrollView != null)
+				_hScrollView.HorizontalScrollBarEnabled = _view.HorizontalScrollBarVisibility == ScrollBarVisibility.Always;
```

Comment:
This will change the behavior of an application where the horizontal scrollbar has been turned "on" by default via a style. Instead of assuming that `ScrollBarVisibility.Default` means "off for horizontal scrollbars" and "on for vertical scrollbars", the value for Default should be cached the first time the visibility is changed.